src/AddItem.js

```diff
@@ -27,6 +27,8 @@ const AddItem = () => {
           name: inputValue,
           frequency: frequency,
           lastPurchased: null,
+          nextPurchase: frequency,
```

Comment:
Are `frequency` & `nextPurchase` the same concept? When a user adds an item she assigns a frequency (soon, not soon, etc., which translates to an integer). Isn't that the value that is updated as a user makes purchases?

If so, these might be redundant.